webpack.dev.ts

```diff
@@ -29,6 +29,7 @@ module.exports = merge(common, {
       '/api/v2': 'http://localhost:8086',
       '/debug/flush': 'http://localhost:8086',
       '/oauth': 'http://localhost:8086',
+      '/health': 'http://localhost:8086',
```

Comment:
For running local development against OSS with `yarn start`, so it can proxy the request to the backend successfully.